### PR TITLE
RDoc-1990 - Elasticsearch ETL fixes

### PR DESCRIPTION
**Pull request by reebhub**

### CODE REVIEW

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown [hunk 1]
```diff
@@ -41,128 +41,148 @@ To begin creating your Elasticsearch ETL task:
 !["Define Elasticsearch ETL task"](images/elasticsearch-etl-define-task.png "Define Elasticsearch ETL task")
 
 1. **Task Name** (Optional)  
-   * Choose a name for your task  
-   * If no name is provided, the cluster will create a name based on the defined connection string (e,g, *ElasticSearch ETL to ElasticConStr*).  
+   * Enter a name for your task  
+   * If no name is provided, the cluster will create a name based on the defined connection string,
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:45]: the cluster => the server

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown [hunk 2]
```diff
@@ -41,128 +41,148 @@ To begin creating your Elasticsearch ETL task:
...
+  to define here an index by the name of `orders`.  
+* The index name entered here must be all lower case, as required by Elasticsearch.  
+  E.g. `orders`
+* The index name used in the transformation script command can be either lower 
+  or upper-case.  
+  E.g. both `loadToOrders` and `loadToorders` are permitted.  
+
+#### 3. Document ID Property Name
+
+* Enter the name of a transformation script property that contains `id(this)`.
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:98]: a transformation => the transformation

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown [hunk 3]
```diff
@@ -41,128 +41,148 @@ To begin creating your Elasticsearch ETL task:
...
+* The index name used in the transformation script command can be either lower 
+  or upper-case.  
+  E.g. both `loadToOrders` and `loadToorders` are permitted.  
+
+#### 3. Document ID Property Name
+
+* Enter the name of a transformation script property that contains `id(this)`.  
+  This property will be created in each generated Elasticsearch document, and 
+  allow the ETL task to recognize the documents by their original RavenDB IDs 
+  even when they are hosted by Elasticsearch.
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:101]: even when they are hosted by Elasticsearch.  
=>
when they are hosted in Elasticsearch.

(remove even, by => in)

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown [hunk 4]
```diff
@@ -41,128 +41,148 @@ To begin creating your Elasticsearch ETL task:
...
+     a document to the Elasticsearch destination.  
+     E.g., `var orderData` in the above example.  
+   * Make sure that one of the properties of the document object 
+     is given the value `id(this)`.  
+     {NOTE: }
+     The ETL task will use this property [to identify](../../../../studio/database/tasks/ongoing-tasks/elasticsearch-etl-task#document-id-property-name) 
+     documents that reside on the Elasticsearch destination by their source 
+     RavenDB document ID.  
+     {NOTE/}
+   * Use the loadTo\<indexName\> method to pass the document object
```
Comment (Danielle9897) [inline Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:182]: make `loadTo\<indexName\>` blue

**Review by Danielle9897 — CHANGES_REQUESTED** [review]

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:45]: done

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:98]: done

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:101]: done

File: Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown (hunk 4 above)
Comment (reebhub) [reply Documentation/5.3/Raven.Documentation.Pages/studio/database/tasks/ongoing-tasks/elasticsearch-etl-task.markdown:182]: done

**Review by Danielle9897 — APPROVED** [review]

**Review by Danielle9897 — APPROVED** [review]

**Pull request merged**